github/teams.go

```diff
@@ -121,6 +123,7 @@ func (s *TeamsService) GetTeam(ctx context.Context, team int64) (*Team, *Respons
 
 // NewTeam represents a team to be created or modified.
 type NewTeam struct {
+	NodeID       *string  `json:"node_id,omitempty"`
```

Comment:
As far as I can tell, this field doesn't need to be added to `NewTeam`. `NewTeam` is used when creating or editing a team, but neither of those two endpoints accept a `node_id` parameter:

- https://developer.github.com/v3/teams/#create-team
- https://developer.github.com/v3/teams/#edit-team

Only the response does, but struct `Team` corresponds to that.